.github/CODEOWNERS

```diff
@@ -57,4 +57,5 @@
 /content                                        @bdunne
 /config                                         @jrafanie
 /db                                             @agrare @carboni @bdunne
+/lib/manageiq/reporting                         @panspagetka @jrafanie
```

Comment:
Add @PanSpagetka and I as the reviewers for this directory from a code and code loading side.